packages/web/src/scss/tools/_form-fields.scss

```
@@ -153,6 +153,16 @@
     &:focus-within {
         background-color: form-fields-settings.$input-background-color-state-active;
     }
+
+    @each $breakpoint-name, $breakpoint-value in tokens.$breakpoints {
+        @include breakpoint.up($breakpoint-value) {
+            border-radius: var(
```

This works and it's fine. However, we (now more than before, so you may find both approaches in our code base) prefer using CSS variables to define variable (sic!) properties.

For example, line 138:

```scss
@mixin box-field-input($field-name) {
  // …
  border-radius: var(--#{tokens.$css-variable-prefix}#{$field-name}-input-border-radius);
  // …
}
```

and here:

```suggestion
            --#{tokens.$css-variable-prefix}#{$field-name}-input-border-radius: var(
```

See `Button` styles for inspiration.